packages/itwinui-react/src/core/ColorPicker/ColorPalette.tsx

```diff
@@ -45,7 +45,8 @@
  * </ColorPalette>
  */
 export const ColorPalette = React.forwardRef((props, ref) => {
-  const { colors, label, className, children, ...rest } = props;
+  const uid = useId();
```

Comment:
might be able to avoid this weird warning by using a different name than `uid`

or even better, remove the variable name altogether

```diff
- const uid = useId();
- const { colors, label, className, children, id = uid, ...rest } = props;
+ const { colors, label, className, children, id = useId(), ...rest } = props;
```

Reply:
Done

Reply:
actually that breaks rule of hooks, so should be reverted. try just using a different variable name, e.g. `stableId` instead of `uid`.